KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -94,9 +98,10 @@ public static Arkivmelding GetArkivmelding(OppdaterSaksmappe input)
                 {
                     var dokumentbeskrivelse = new Dokumentbeskrivelse
                     {
+                        Dokumenttype = "TEST", //TODO Hva skal inn her?
```

Comment:
Hva skal være standard? Eller skal det komme som inn-parameter i OppdaterSaksmappe?

Reply:
Her bør du gå for kode fra ArkivLett.

ArkivLett har kommet med anbefaling for koder.
Men koden bør absolutt komme fra klienten og ikke lages automatisk.

Reply:
Da blir det `input.nyUtgaaendeJournalpost.hoveddokument.dokumenttype.kodeverdi` som ser mest riktig ut.